storage/innobase/trx/trx0trx.cc

```diff
@@ -1356,17 +1355,17 @@ static void trx_start_low(
                           std::memory_order_relaxed);
   }
 
-#if 0
 #ifdef WITH_WSREP
+#if 0
```

Comment:
Maybe remove this dead code? Leave comment if still relevant.

Reply:
Done. Removed.